src/components /Sighting.js

```diff
@@ -19,6 +47,38 @@ const Sighting = () => {
 
   let navigate = useNavigate();
 
+  useEffect(() => {
+    axios
+      .get(`${serverURL}/${sightingID}/comments`)
+      .then((response) => setComments(response.data));
+  }, [sightingID]);
+
+  const commentList = comments.map((comment) => {
+    return comment.content;
```

Comment:
in your db, is content set as NOT NULL? If not, you might face some empty strings here